src/components/Kv/KvIconButton.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<kv-button class="icon-btn"
+		@click.native.prevent="null"
+		:style="cssVars"
+	>
+		<div class="icon-background">
+			<div class="icon-wrapper icon icon--left">
+				<slot name="icon-left"></slot>
+			</div>
+		</div>
+		<span><slot></slot></span>
+		<div class="icon-wrapper icon icon--right">
+			<slot name="icon-right"></slot>
+		</div>
+	</kv-button>
+</template>
+
+<script>
+import KvButton from '@/components/Kv/KvButton';
+
+export default {
+	components: {
+		KvButton,
+	},
+	props: {
+		theme: {
+			type: String,
+			default: 'ghost',
+		},
+	},
+	computed: {
+		cssVars() {
+			// TODO add other button themes
+			if (this.theme === 'ghost') {
+				// $platinum: #efefef;
+				// $umbreon: #212121;
+				// $white: #fff;
+				// $dark-blue: #006ed3 or 0,100,211
+				return {
+					'--kv-left-icon-background': '#efefef',
+					'--kv-left-icon-color': '#212121',
+					'--kv-left-icon-hover': '#006ed3',
+					'--kv-color-hover': '#fff',
+					'--kv-left-icon-color-active': '0,110,211',
+
+				};
+			}
+			return {};
+		}
+	},
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+.icon-btn {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	flex-wrap: wrap;
+
+	.icon-background {
+		background-color: var(--kv-left-icon-background);
+		border-radius: 50%;
+		height: 2rem;
+		width: 2rem;
+		max-width: 2rem;
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		margin-right: 1rem;
+	}
+
+	.icon-wrapper ::v-deep div {
```

Comment:
Is this "deep" div within the KvIcon or a markup element that could change?

Reply:
Its the KvIcon wrapper.